src/Sarif.Driver/Sdk/ConsoleLogger.cs

```diff
@@ -75,8 +75,8 @@ public void Log(IRuleDescriptor rule, Result result)
             // Note that we can potentially emit many messages from a single result
             WriteToConsole(
                 result.Kind,
-                result.Locations?[0].AnalysisTarget?[0].Uri,
-                result.Locations?[0].AnalysisTarget?[0].Region,
+                result.Locations?[0].AnalysisTarget?.Uri,
```

Comment:
Not your bug but this logger should consult resultFile (if non-null) and then fall back to analysisTarget if required. Filed #145